Project_0.Altis/description.ext

```diff
@@ -55,9 +55,11 @@ class Params {
 };
 
 class CfgFunctions {
+	#include "CivilianPresence\functions.hpp"
```

Comment:
We add functions to CfgFunctions like good boys do!

Reply:
Is easy for me to work with, recompile and copy the function name